docs/GettingStartedDocs/GettingStarted.Windows.md

```diff
@@ -118,100 +127,59 @@ You can change the build settings by with the CMake > Change CMake Settings menu
 option. This opens the [CMakeSettings.json](https://blogs.msdn.microsoft.com/vcblog/2017/08/14/cmake-support-in-visual-studio-customizing-your-environment/)
 file which you can edit and change settings such as the target build location.
 
-By default, VS2017 will build to a dynamically generated folder name:
+By default, Open Enclave SDK will build in the following location:
 ```
-${env.USERPROFILE}\CMakeBuilds\\${workspaceHash}\\build\\${name}
+${workspaceRoot}\build\<configuration-name>
 ```
 For example:
 ```
-C:\Users\username\CMakeBuilds\ca6d6e50-e70d-c836-ac64-910bf7e68090\build\x64-Debug
+C:\openenclave\build\x64-Debug
 ```
 
 ### Building on Windows using Developer Command Prompt
 
-1. From the Start menu, launch the [Developer Command Prompt](
+1. Launch the [x64 Native Tools Command Prompt for VS 2017](
 https://docs.microsoft.com/en-us/dotnet/framework/tools/developer-command-prompt-for-vs)
-2. At the command prompt, use cmake and nmake to build the project:
+Normally this is accessible under the `Visual Studio 2017` folder in the Start Menu.
+2. At the x64 Native Tools command prompt, use cmake and nmake to build the project:
    ```
    C:\> cd C:\openenclave
```

Comment:
When the GettingStarted.md docs were updated, they standardized on other OSS examples:
* if just listing a series of instructions, do not include the prompt (as in this case) to facilitate copy & paste. 
* if showing expected output, include the prompt (as in examples on lines 58 & 69)

Also, you should use '```cmd' to get syntax highlighting in the code block:

```cmd
cd C:\openenclave
mkdir build\x64-Debug
cd build\x64-Debug
cmake -G Ninja -DBUILD_ENCLAVES=1 ../..
ninja
```

Apply here and the testing examples below

Reply:
Nice. Done.